.github/workflows/release.yml

```diff
@@ -5,6 +5,11 @@ on:
     branches:
       - main
 
+permissions:
+  id-token: write # Required for OIDC
+  contents: write # Required for changesets to create commits and tags
+  pull-requests: write # Required for changesets to open/update release PRs
```

Comment:
If we don't set this two permission, we get: https://github.com/scaffold-eth/scaffold-ui/actions/runs/20294293430/job/58284663109